src/commands/transaction/multisigmodification.ts

```diff
@@ -35,19 +32,20 @@ import {
 } from 'symbol-sdk'
 import {command, metadata, option} from 'clime'
 import chalk from 'chalk'
+import {DeltaResolver} from '../../resolvers/delta.resolver'
 
 export class CommandOptions extends AnnounceAggregateTransactionsOptions {
     @option({
         flag: 'R',
-        description: '(Optional) Number of signatures needed to remove a cosignatory. ',
-        default: 0,
+        description: 'Number of signatures needed to remove a cosignatory. ' +
+            'If the account already exists, enter the relative amount of change.',
```

Comment:
enter the relative number of cosigners to add or remove?

I think that "the relative amount of change" is not grammatically correct, since we are talking of numbers, and change should be whether in plural form or prefixed with "the"